### ISSUE TITLE
Add custom vendor list of items

### ISSUE THREAD

**Issue by Phr33d0m** [Phr33d0m/NW-Profession-Bot#174]
As per [this idea](https://greasyfork.org/en/forum/discussion/4886/x) it'd be nice to have a custom vendor list option somewhere.

**Comment by noonereally** [comment]
Dangerous, the exclusion filter should be done very carefully or a tiny regex mistake and half the inventory vendored.

A safer option you can do is to use the same code from the item tracker and vendor items without regex, but exact match. And the item tracker settings should be moved to the account level anyway.

**Comment by BigRedBot** [comment]
I have been thinking that an option to vendor/discard all character bound items would be nice. But since this option is so dangerous, it should only be allowed to be enabled per character, and a confirmation asking you to write the character's name should be completed before enabling this option, for each and every character.

**Comment by gomsnothereman** [comment]
for alt accounts, this does not sound quite so scary.. but for a main _shiver_ i think brent has a great idea with making it as difficult as possible to turn on, and also forcing it on a character by character basis..

**Comment by BigRedBot** [comment]
There may need to be some exclusions to that though. Such as the one time only boxes you get for leveling up. I am pretty sure those are character bound, and should not be able to be discarded. Perhaps anything that opens up should be excluded from this vendor/discard all character bound items option. That may make it a lot simpler.

But for the most part, anything that can not be opened up that is character bound, would be useless to keep if it is just on an alt.

**Comment by Phr33d0m** [comment]
> without regex, but exact match

I like this.

**Comment by Pazarion** [comment]
I all time have problems with OverBags,  version 4.4.2 helped a lot but need add some modification. I Offer add possibility to sell [Lesser Mark of Potency][Lesser Mark of Power][Lesser Mark of Stability][Lesser Mark of Union][Minor Power Stone][Minor Union Stone][Minor Stability Stone] in this way we can clean 14 inventory slots ( bound and unbound items). + Due- to SCA runs I have unidentified useless items in inventory  so can you add possibility to sell this items automatically too.

**Comment by Phr33d0m** [comment]
Closing as per #247
